### PR TITLE
[TS] LPD-25100

### PR DESCRIPTION
**Pull request by ericyanLr**
Hi Team,

**Ticket:** https://liferay.atlassian.net/browse/LPD-25100
**Previous PR:** https://github.com/liferay-core-infra/liferay-portal/pull/6950

As discussed, I have added a test for this issue.

**Original PR Description**
When the InvokerFilterHelper class reads in the initial parameters, it checks for properties namespaced with init.params. However, the properties are set as init-params throughout the portal, and I could find nothing online using the . notation, so I assume it is incorrect. This leads to certain properties not being used.

It's possible init.param is correct, and all the filters need to be adjusted instead, however, I've found no examples of this online, which leads me to believe init-param is correct.

**Analysis**
Based on our analysis, we agree that `init-param` should be the correct notation.

----

**Additional Note**
For this integration test, I decided to add it to the module: `portal-servlet-test`, as it looked the most appropriate. But, notice that the original InvokerFilter class resides in the package: `com.liferay.portal.kernel.servlet.filters.invoker`. 

I decided to instead put the test in the package: `com.liferay.portal.servlet.filters.invoker.test` to workaround a SourceFormatter rule. 

If we try to put it in the package: `com.liferay.portal.kernel.servlet.filters.invoker.test`, SourceFormatter would complain.
- Reference: https://github.com/liferay/liferay-portal/blob/master/modules/util/source-formatter/src/main/resources/documentation/check/java_package_path_check.markdown#exceptions

Please let me know if you have any questions.
Thanks!

### CODE REVIEW

**Comment by ericyanLr** [comment]
ci:test:sf

**Comment by ericyanLr** [comment]
ci:test:relevant

**Comment by ericyanLr** [comment]
ci:report:2567467799

File: modules/apps/portal/portal-servlet-test/src/testIntegration/java/com/liferay/portal/servlet/filters/invoker/test/InvokerFilterTest.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+public class InvokerFilterTest {
+
+	@ClassRule
+	@Rule
+	public static final AggregateTestRule aggregateTestRule =
+		new LiferayIntegrationTestRule();
+
+	@Test
+	public void testOSGIComponentFilterInitParams() {
+		BundleContext bundleContext = SystemBundleUtil.getBundleContext();
```
Comment (dantewang) [inline modules/apps/portal/portal-servlet-test/src/testIntegration/java/com/liferay/portal/servlet/filters/invoker/test/InvokerFilterTest.java:41]: @ericyanLr If it's an integration test, I think you can just use the test module's bundle context. A random example: `ContentDashboardItemFilterProviderRegistryTest`.

**Review by dantewang — CHANGES_REQUESTED** [review]

**Comment by dantewang** [comment]
Hi @ericyanLr what is the status of this PR? Did it pass the tests? We may need to push it faster as the GS/Support for a customer wants to have this in the hotfix being handed to the customer.

cc @tinatian

**Comment by dantewang** [comment]
ci:report:2567810357

**Comment by dantewang** [comment]
@ericyanLr FYI, as per requested, I created [a branch](https://github.com/dantewang/liferay-portal-ee/tree/2023.q4.6-LPD-25100) with the commits cherry-picked on top of 2023.q4.6 and added it to [patcher](https://patcher.liferay.com/group/guest/patching/-/osb_patcher/fixes/259903002) portal.

**Comment by tinatian** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/150324

**Comment by ericyanLr** [comment]
@dantewang @tinatian ,

Thank you for working on this and for the suggestions and review, I forgot to follow-up on this PR yesterday.

**Pull request closed without merging**